cli/tests/fixtures/test_package/buff.toml

```diff
@@ -0,0 +1,6 @@
+[package]
```

Comment:
i think it'll be very cool to do a full package spec, even if we are not intend to support everything from the get-go.